lib/vty.c

```diff
@@ -95,7 +95,7 @@ static int no_password_check = 0;
 /* Integrated configuration file path */
 char integrate_default[] = SYSCONFDIR INTEGRATE_DEFAULT_CONFIG;
 
-static int do_log_commands = 0;
+static bool do_log_commands = false;
```

Comment:
Per checkpatch warns, statics should not be initialized as a matter of style

Reply:
fixed